locals.tf

```diff
@@ -41,6 +41,7 @@ locals {
       tag_filter_any = repo.parameters != null ? (repo.parameters.tag_filter_any != null ? repo.parameters.tag_filter_any : "") : ""
       tag_filter_all = repo.parameters != null ? (repo.parameters.tag_filter_all != null ? repo.parameters.tag_filter_all : "") : ""
       recursive      = true
+      dry_run        = repo.parameters != null ? (repo.parameters.dry_run != null ? repo.parameters.dry_run : false) : false
       filter         = repo.parameters != null ? "grace-${repo.parameters.grace != null ? repo.parameters.grace : "0"}-keep-${repo.parameters.keep != null ? repo.parameters.keep : "0"}-tag_filter-${repo.parameters.tag_filter != null ? repo.parameters.tag_filter : "no"}-tag_filter_any-${repo.parameters.tag_filter_any != null ? repo.parameters.tag_filter_any : "no"}-tag_filter_any-${repo.parameters.tag_filter_any != null ? repo.parameters.tag_filter_any : "no"}" : "delete-all-untagged-images-recursive"
```

Comment:
```suggestion
      filter         = repo.parameters != null ? "grace-${repo.parameters.grace != null ? repo.parameters.grace : "0"}-keep-${repo.parameters.keep != null ? repo.parameters.keep : "0"}-tag_filter-${repo.parameters.tag_filter != null ? repo.parameters.tag_filter : "no"}-tag_filter_any-${repo.parameters.tag_filter_any != null ? repo.parameters.tag_filter_any : "no"}-tag_filter_any-${repo.parameters.tag_filter_any != null ? repo.parameters.tag_filter_any : "no"}" : "delete-all-untagged-images-recursive${repo.parameters.dry_run != null ? "${repo.parameters.dry_run ? "-dry_run" : ""}" : ""}"
```

Reply:
Updating the filter (that will be used in the name of the job) to add `dry-run` if set to true.

Reply:
@anouarchattouna I updated the filter with `dry_run` as requested, but I put it only if `parameters` is set

Reply:
Thanks!